Sources/Plasma/PubUtilLib/plPipeline/plGBufferGroup.cpp

```diff
@@ -439,24 +417,24 @@ void    plGBufferGroup::Read( hsStream *s )
     for( i = 0; i < count; i++ )
     {
         temp = s->ReadLE32();
-        fIdxBuffCounts.Append( temp );
-        fIdxBuffStarts.Append(0);
-        fIdxBuffEnds.Append(-1);
+        fIdxBuffCounts.push_back(temp);
+        fIdxBuffStarts.push_back(0);
+        fIdxBuffEnds.push_back(-1);
```

Comment:
Whenever you see `push_back` in a known-count loop, it's a good idea to `reserve` the space in the vector first...